MerlinAU.sh

```diff
@@ -4218,25 +4227,30 @@ Please manually update to version $minimum_supported_version or higher to use th
     fi
 
     ##------------------------------------------##
-    ## Modified by ExtremeFiretop [2024-Feb-03] ##
+    ## Modified by ExtremeFiretop [2024-May-21] ##
     ##------------------------------------------##
-    if [ -f "sha256sum.sha256" ] && [ -f "$firmware_file" ]; then
+
+    # Fetch the latest checksums from ASUSWRT-Merlin website
+    checksums=$(curl -s https://www.asuswrt-merlin.net/download | sed -n '/<pre>/,/</pre>/p' | sed -e 's/<[^>]*>//g')
```

Comment:
You might consider adding some retry options to make curl more resilient when fetching. For example, amtm uses `-fsNL --connect-timeout 10 --retry 3 --max-time 12` options.

Reply:
> You might consider adding some retry options to make curl more resilient when fetching. For example, amtm uses `-fsNL --connect-timeout 10 --retry 3 --max-time 12` options.

Great idea, I think our standard is 5 retries within the script. I'll handle this now. Thanks for the suggestions as always!

Reply:
@dave14305 

Done. Thanks for the suggestion!